site/docs/types/simple_logical_types.md

```diff
@@ -6,22 +6,19 @@ To minimize type explosion, the project currently follows the guideline that a l
-| timestamp | A timestamp with microsecond precision                       | Timestamp&lt;MICROSECOND&gt; | timestamp      | TimestampType | timestamp(6)           |
-| date            | A date       | Date&lt;MILLISECOND&gt;      | date           | DateType      | Date                   |
-| time      | A time with microsecond precision since the beginning of any day. Maximum value is one day. | Time&lt;MICROSECOND;64&gt;   | time           | time(6)       | time(6)                |
-| interval_year   | Interval day to month                                        | INTERVAL&lt;YEAR_MONTH&gt;   | -              | -             | Interval year to month |
-| interval_day    | Interval day to second                                       | INTERVAL&lt;DAY_TIME&gt;     | -              | -             | Interval day to second |
-
-#### Discussion points
-
-* How do we ensure Substrait is adoptable by different communities while avoiding type explosion? Is it important to avoid type explosion? Current proposal is to avoid types unless they exist in at least two projects.
-* Does it make sense to support user defined types?
\ No newline at end of file
+| string          | A string of text. [0..2,147,483,647] bytes in length. String is encoded using UTF8 encoding. | Utf8                   | string         | StringType    | varchar (no len)       |
+| binary          | A binary value. [0..2,147,483,647] bytes in length. | Binary                 | binary         | BinaryType    | Varbinary              |
+| timestamp | A naive timestamp with microsecond precision that cannot be mapped to a moment on the timeline. Similar to naive datetime in Python. Supports range of [1000-01-01 00:00:00.000000..9999-12-31 23:59:59.999999] | timestamp&lt;MICROSECOND&gt; | timestamp      | TimestampType | timestamp(6)           |
+| timestamp_tz | A timestamp with microseconds precision that is mapped to an instant in time. Similar to aware datetime in Python. Supports a range of [1000-01-01 00:00:00.000000..9999-12-31 23:59:59.999999] UTC. | timestamp&lt;micro;utc&gt; | timestamptz | - | timestamp(6) with time zone |
+| date            | A date. Range of [1000-01-01..9999-12-31]. | Date&lt;MILLISECOND&gt;      | date           | DateType      | Date                   |
+| time      | A time with microsecond precision since the beginning of any day. Range of [0..86,399,999,999] microseconds. | Time&lt;MICROSECOND;64&gt;   | time           | time(6)       | time(6)                |
+| interval_year   | Interval year to month. Supports a range of [0..120000] months. | INTERVAL&lt;YEAR_MONTH&gt;   | -              | -             | Interval year to month |
+| interval_day    | Interval day to second with microsecond precision. Supports a range of  [0..3,650,000] days and [0..86,399,999,999] microseconds. | INTERVAL&lt;DAY_TIME&gt;     | -              | -             | Interval day to second |
```

Comment:
Why can't intervals be negative?  I know Arrow & postgres support negative intervals.  I'm not sure about Trino but I would suspect it is allowed.

Reply:
Good point on the interval ranges. I really struggled with those ranges. Frankly, I just really hate interval types :)

I've pushed an update. LMK if you think it makes sense.